changelog.json

```
@@ -1,5 +1,12 @@
 {
   "releases": {
+    "2.1.1-beta1": [
+      "[Added] WSL Support - #7821 #7891. Thanks @ADustyOldMuffin @say25!",
+      "[Fixed] Handle \"bad revision\" error when starting a rebase - #7871",
```

We should be consistent with formatting, ex:
```suggestion
      "[Fixed] Incorrectly shown \"bad revision\" error when starting a rebase - #7871",
```
or something. Not married to this phrasing.

---

what about this? since we don't really prevent the error we just communicate it better? @nerdneha 
```suggestion
      "[Improved] Error handling when trying to rebase onto an invalid ref - #7871",
```

---

👍 lgtm!